### PR TITLE
feat(431): Add executor queue feature file

### PR DESCRIPTION
**Pull request by kumada626**
Add cucumber feature file of executor queue.
How does it look?

Related:#431

### CODE REVIEW

**Comment by kumada626** [comment]
Fixed validator.output.json because of updating data-schema.

File: features/executor-queue.feature [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    Users want their build to enqueue for later execution if all the execution engines are
+    offline or full, so that they do not have to worry about forgotten builds.
+
+    Screwdriver should be pushing planned builds into a queue. And all available executors
+    should be listening on that queue and taking items off when they have capacity.
+
+    Rules:
+        - Executors can determine if they are capable of running a new build.
+        - Multiple instances of the same executor can connect to the queue.
+        - Multiple executor types can connect to the same queue.
```
Comment (FenrirUnbound) [inline features/executor-queue.feature]: It looks like these rules were inherited from feature #431. I don't see this specific scenario being covered. Is it safe to assume that this scenario won't be covered yet?

Comment (petey) [inline features/executor-queue.feature]: There should probably be a test for when an executor `can not` handle a new build.

Comment (kumada626) [inline features/executor-queue.feature]: Added a test case when an executor can not handle a new build, and fixed all tests cope with multiple executor types.

File: features/executor-queue.feature [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    Scenario Outline: Push a build into the queue
+        Given an executor type is <type>
+        When a new build is created
+        Then that build is pushed into the queue
+        
+        Examples:
+            | type   |
+            | k8s    |
+            | docker |
+            | j5s    |
```
Comment (stjohnjohnson) [inline features/executor-queue.feature]: On a side note (not a blocker here) I really want to rename `j5s` into `jenkins`

Comment (kumada626) [inline features/executor-queue.feature]: Renamed j5s into jenkins.

**Pull request merged**